register-autonomoustuff-repository/action.yaml

```diff
@@ -0,0 +1,26 @@
+name: register-autonomoustuff-repository
+description: Register AutonomouStuff repositry into rosdep
```

Comment:
Could you write a brief README.md for this action, please? :pray: 

This is an example.
https://github.com/tier4/github-actions/tree/main/ros-modified-packages

Reply:
Also, please follow the decision that will be determined here.
https://github.com/autowarefoundation/autoware-github-actions/pull/4#discussion_r764468366

Reply:
Fixed at https://github.com/autowarefoundation/autoware-github-actions/pull/6/commits/34bb9899aad3e9c9823c5b2010a01b97c79f4b50